crates/ln-dlc-node/src/tests/load.rs

```diff
@@ -0,0 +1,177 @@
+use crate::ln::app_config;
+use crate::node::Node;
+use crate::node::NodeInfo;
+use crate::node::PaymentMap;
+use crate::tests::init_tracing;
+use crate::tests::wait_until;
+use anyhow::Context;
+use anyhow::Result;
+use coordinator::Coordinator;
+use coordinator::Direction;
+use dlc_manager::subchannel::SubChannelState;
+use dlc_manager::Storage;
+use std::borrow::Borrow;
+use std::sync::Arc;
+use std::time::Duration;
+
+mod coordinator;
+
+const ESPLORA_ORIGIN_PUBLIC_REGTEST: &str = "http://35.189.57.114:3000";
+
+#[cfg(feature = "load_tests")]
+#[tokio::test]
+async fn single_app_many_positions_load() {
```

Comment:
I would consider moving it to an example eventually, because we could use `clap` to pass in arguments, e.g. the amount or even whether to try against local coordinator.

Reply:
I think it's a really good idea to create a dedicated binary for this. It's not really a test that should be treated like the others.

But it's not really an example either. I think it could just be a separate crate, right? Same goes for `fund`.

Reply:
I don't mind having them as crates; it's also better than as a test as it gives us the same benefits. 
an example for me was a nice way of preventing its compilation during release etc. otherwise they'll need to get *always* compiled along with the workspace - I thought you cared about this part after seeing the feature flag, hence the "example" suggestion.